Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
Ah, that's nice. Let's hope it works with 2.61, which I would like to stick to in support of older CentOS (and probably others). Let's include it in a separate step. It is OK to remove the current -Werror, as it right now does not affect configure, and then re-add the functionality.